### DISCUSSION TITLE
lasergrbl w/ortur rotary

### DISCUSSION THREAD

**Discussion by ortur2pronewbie** [arkypita/LaserGRBL#1660] [General]
hello,

newbie to lasering but want to start using my ortur rotary. i have only been practicing with lasergrbl but want to know if it is possible to use the rotary with lasergrbl or do i have to start learning lightburn? thank you

**Comment by arkypita** [comment]
With rotary roller engravingn cylindrical surface is the same as to flat surface, so you can use LaserGRBL in the same way.
